ts/services/search-service.ts

```diff
@@ -2,11 +2,14 @@ import { PathVariable } from '../xpath-extractinator';
 import * as $ from 'jquery';
 
 export class SearchService {
-    public getAllResults(variables: PathVariable[]) {
+    /**
+     * @param isAnalysis Whether these results are retrieved for analysis: a higher limit can be set for this in the configuration.
+     */
+    public getAllResults(variables: PathVariable[], isAnalysis = false) {
         // TODO: read URL from config
```

Comment:
Does an issue exist for this todo?

Reply:
Well vaguely, when introducing/rewriting to use Angular 4 (#45) this would most probably pop up. You could then of course more easily use things like dependency injection for the configuration.